src/components/ChainNameAndIcon.tsx

```diff
@@ -1,31 +1,19 @@
 import { getChainIcon } from "@/constants";
 import type { ChainId, ChainName } from "@shared/types";
-import styled from "styled-components";
 
 interface Props {
   chainName: ChainName;
   chainId: ChainId;
 }
 export function ChainNameAndIcon({ chainName, chainId }: Props) {
-  const chainIcon = getChainIcon(chainId);
+  const ChainIcon = getChainIcon(chainId);
```

Comment:
with this new setup svgs can be used directly as components and they can be sized with tailwind classes